python/openmldb/driver.py

```diff
@@ -59,6 +59,13 @@ def dropDB(self, db):
             return True, "ok"
         else:
             return False, status.msg
+        
+    def getAllTables(self):
+        if not self.sdk:
+            return False, "please init driver first"
+        status = sql_router_sdk.Status()
+        all_table = self.sdk.GetAllTables(status)
```

Comment:
Check status and only return all_table if success.

Reply:
updated.